core/src/main/java/org/kiwix/kiwixmobile/core/page/PageFragment.kt

```diff
@@ -84,15 +90,14 @@ abstract class PageActivity : OnItemClickListener, BaseActivity() {
       }
     }
 
-  override fun onCreateOptionsMenu(menu: Menu): Boolean {
-    menuInflater.inflate(R.menu.menu_page, menu)
+  override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+    inflater.inflate(R.menu.menu_page, menu)
     val search = menu.findItem(R.id.menu_page_search).actionView as SearchView
     search.queryHint = searchQueryHint
     search.setOnQueryTextListener(SimpleTextListener {
       pageViewModel.actions.offer(Action.Filter(it))
     })
     pageViewModel.state.observe(this, Observer(::render))
```

Comment:
`this` is incorrect for an observe in a fragment, you want `viewLifecycle` I would think or something similarly named

Reply:
`viewLifecycleOwner`?